### ISSUE TITLE
Organice eats strikethrough text in check list

### ISSUE THREAD

**Issue by MisterY** [200ok-ch/organice#640]
**Describe the bug**
Organice "eats" the parts of the note that are between pluses (strike through). 

**To Reproduce**
1. Add a header as specified below 
2. Access the file in Organice
3. Check or uncheck the checkbox
4. The parts between the +plus signs+ are gone

```
* TODO Stuff
- [ ] +something+
```

**Expected behavior**
The strike through formatting should remain along with the text 

**Smartphone:**
 - Device: Xiaomi Mi 9T
 - OS: LineageOS 17.1
 - Browser: Firefox Beta

**Comment by MisterY** [comment]
Interestingly, I can no longer reproduce this either with the copy of the original file nor the original.

**Comment by schoettl** [comment]
Thanks for the report @MisterY. I slightly changed the bug description and added a minimal example.

The striked text is gone when checking/unchecking a checkbox.

**Comment by munen** [comment]
This got solved somewhere along the way. Here's how the example from above looks in Emacs and in current organice:

![image](https://user-images.githubusercontent.com/505721/174816568-84b563ae-a6be-4ebe-b748-81f68a652aac.png)

Imo it behaves correctly. Likely somebody fixed it and forgot to close the issue.

Thank you for the initial report :pray:

**Comment by chasecaleb** [comment]
This is still a bug and I can reproduce it. @munen you need to click the checkbox to trigger the bug (you don't have it checked in your screenshot).

Here are the exact steps that I took:

1. In Organice, create this heading:
     ```org
     * TODO Stuff
     - [ ] +something+
     ```
2. Check the checkbox
3. Wait for sync
4. In Organice it looks correct at this point, but if you look at the file in your sync backend (GitLab repo, dropbox, whatever) you'll see the strikethrough content is gone.
5. Refresh Organice and you'll see the strikethrough content is gone there as well.

However this isn't just an issue with strikethroughs, it affects markup in general such as italics and markdown -- so I think this issue can stay closed as a duplicate of #376. I just want to make sure we're on the same page that this is still an issue. You can also refer to #796 which I created -- it's a duplicate of #376, but it has some more repro info and screenshots.

**Comment by munen** [comment]
@chasecaleb My apologies :pray: 

You are right, of course! I'll leave this issue closed in favour of https://github.com/200ok-ch/organice/issues/376 which has linked to this issue.

Thank you for double checking :+1:
